app/components/Buttons.tsx

```diff
@@ -0,0 +1,17 @@
+export default function PrimaryButton() {
+  return (
+    <button className="bg-gray-800 rounded-3xl py-1 px-8 m-1 text-white hover:bg-emerald-50 hover:text-gray-800 uppercase font-semibold">Join</button>
+  )
+}
+export function SecondaryButton() {
```

Comment:
We could merge all these buttons into one and conditional apply the classes based on a prop `buttonType: 'warm' | 'primary' | 'secondary'`.

This way, you can quickly change the button type by updating the prop value:

```jsx
<Button buttonType="primary">Join</Button>
```

Reply:
This is done per suggestion!